package.json

```diff
@@ -18,52 +18,74 @@
   },
   "scripts": {
     "build": "ember build --environment=production",
-    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*",
+    "lint": "concurrently \"npm:lint:*(!fix)\" --names \"lint:\"",
+    "lint:css": "stylelint \"**/*.css\"",
+    "lint:css:fix": "concurrently \"npm:lint:css -- --fix\"",
+    "lint:fix": "concurrently \"npm:lint:*:fix\" --names \"fix:\"",
     "lint:hbs": "ember-template-lint .",
-    "lint:js": "eslint .",
+    "lint:hbs:fix": "ember-template-lint . --fix",
+    "lint:js": "eslint . --cache",
+    "lint:js:fix": "eslint . --fix",
     "start": "ember serve",
-    "test": "npm-run-all lint:* test:*",
+    "test": "concurrently \"npm:lint\" \"npm:test:*\" --names \"lint,test:\"",
     "test:ember": "ember test",
     "test:ember-compatibility": "ember try:each"
   },
   "dependencies": {
-    "ember-cli-babel": "^7.22.1",
-    "ember-cli-htmlbars": "^5.3.1"
+    "@babel/core": "^7.22.10",
+    "ember-auto-import": "^2.6.3",
+    "ember-cli-babel": "^8.0.0",
+    "ember-cli-htmlbars": "^6.3.0",
+    "ember-simple-auth": "^6.0.0"
```

Comment:
This shouldn't have been move to `dependencies` since it's also a `peerDependency` with a wide range. I think we also implicitly support ember-simple-auth v7 if we remove this dependency again (and consider it a bugfix).

Reply:
Created https://github.com/mu-semtech/ember-mu-login/pull/10 to do just that